internal/apm/php.go

```diff
@@ -84,6 +84,10 @@ type PhpInjector struct {
 	acceptVersion
 }
 
+var appendPathsEnvKeys = map[string]struct{}{
```

Comment:
Only the PHP env var PHP_INI_SCAN_DIR has a concatenation structure using ":", there is no need for the appendPathEnvKeys structure here.  All other env variables should be handled as they have been.